packages/connectivity/src/components/ConnectivityPage.tsx

```diff
@@ -12,12 +12,14 @@ export interface IConnectivityPage
   className?: string;
   headClassName?: string;
   contentClassName?: string;
+  fitContent?: boolean;
 }
 
 export const ConnectivityPage: FC<IConnectivityPage> = ({
   children,
   className,
   rootPath = defaultRootPath,
+  fitContent,
```

Comment:
This should be the default

Reply:
It'll break all components for all customers who use it already.
To support this parameter, a parent container should have a determined height.